### PR TITLE
k11_pro readme.md updated: fix instruction errors

### PR DESCRIPTION
**Pull request by mindeng**
Corrected the compilation & flashing instructions in README.md for k11_pro.

只是更正了 k11_pro 的 README.md 文件中的指令错误，请帮忙 review 一下，谢谢。

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [x] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/keychron/k11_pro/ansi_encoder/white/keymaps/via/keymap.c
Comment (adophoxia) [inline keyboards/keychron/k11_pro/ansi_encoder/white/keymaps/via/keymap.c:1]: There a reason there's changes in this file when the title of the PR is:
> readme.md updated: fix instruction errors
?

File: keyboards/keychron/k11_pro/ansi_encoder/white/keymaps/via/keymap.c [hunk 1]
```diff
@@ -27,7 +27,7 @@ enum layers{
 // clang-format off
 const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
     [MAC_BASE] = LAYOUT_69_ansi(
-        KC_ESC,  KC_1,     KC_2,     KC_3,    KC_4,    KC_5,    KC_6,     KC_7,    KC_8,    KC_9,    KC_0,     KC_MINS,  KC_EQL,   KC_BSPC,          KC_MUTE,
+        QK_GESC,  KC_1,     KC_2,     KC_3,    KC_4,    KC_5,    KC_6,     KC_7,    KC_8,    KC_9,    KC_0,     KC_MINS,  KC_EQL,   KC_BSPC,          KC_MUTE,
```
Comment (adophoxia) [inline keyboards/keychron/k11_pro/ansi_encoder/white/keymaps/via/keymap.c]: Why the change to `QK_GESC`?

File: keyboards/keychron/k11_pro/readme.md [hunk 2]
```diff
@@ -10,13 +10,13 @@ A customizable 65% and ergonomic keyboard.
...
-    make keychron/k11_pro/ansi_encoder:default
+    make keychron/k11_pro/ansi_encoder/rgb:default
+    make keychron/k11_pro/ansi_encoder/white:default
 
 Flashing example for this keyboard ([after setting up the bootloadHID flashing environment](https://docs.qmk.fm/#/flashing_bootloadhid))
 
-    make keychron/k11_pro/ansi:default:flash
-    make keychron/k11_pro/ansi_encoder:default:flash
+    make keychron/k11_pro/ansi_enoder/rgb:default:flash
+    make keychron/k11_pro/ansi_enoder/white:default:flash
```
Comment (adophoxia) [inline keyboards/keychron/k11_pro/readme.md]: Spelling issue
```suggestion
    make keychron/k11_pro/ansi_encoder/rgb:default:flash
    make keychron/k11_pro/ansi_encoder/white:default:flash
```

**Review by adophoxia — CHANGES_REQUESTED** [review]

File: keyboards/keychron/k11_pro/ansi_encoder/white/keymaps/via/keymap.c (hunk 1 above)
Comment (mindeng) [reply keyboards/keychron/k11_pro/ansi_encoder/white/keymaps/via/keymap.c]: Sorry, my mistake, this should be my local modification ... I will remove this commit from the PR.

Comment (mindeng) [reply keyboards/keychron/k11_pro/ansi_encoder/white/keymaps/via/keymap.c]: done, it has been removed now.

File: keyboards/keychron/k11_pro/readme.md (hunk 2 above)
Comment (mindeng) [reply keyboards/keychron/k11_pro/readme.md]: Fixed
